modules/packages/LinearAlgebra.chpl

```diff
@@ -1857,10 +1857,10 @@ proc solve_triu(const ref U: [?Udom] ?eltType, const ref b: [?bdom] eltType) {
 
      Arrays with any offset are supported, and ``x`` inherits the indexing.
 */
-proc solve(A: [?Adom] ?eltType, ref b: [?bdom] eltType) {
+proc solve(const ref A: [?Adom] ?eltType, const ref b: [?bdom] eltType) {
```

Comment:
Wouldn't it be better to use an `in` intent so that if `b` is dead at the callsite, we could reuse its storage rather than creating a new array `y`?  Or would that not work for reasons I'm overlooking?

Reply:
How do I look at this in full please?

Reply:
Damian - you should be able to see the full function by clicking on "View reviewed changes".  If you're looking for the full file, after the above step you will want to click on the three dots in the upper right, then select "View File"